ember/app/ui/cases/detail/work-items/edit/form/controller.js

```diff
@@ -4,9 +4,30 @@ import { inject as service } from "@ember/service";
 
 export default class CasesDetailWorkItemsEditFormController extends Controller {
   @service router;
+  @service notification;
+  @service intl;
 
   @action
   transitionToCaseWorkItems() {
+    /*
+      Display notification to redo when additional-data has been skipped
+      by decision-and-credit but define-amount is being rejected, 
+      leading to no open additional-data-form which would place the workflow in a stuck state
+    */
+    if (
+      this.model.workItem.task.slug === "define-amount" &&
+      this.model.workItem.case.workItems.edges
+        .findBy("node.task.slug", "decision-and-credit")
+        .node.document.answers.edges.findBy(
+          "node.question.slug",
+          "decision-and-credit-decision"
+        ).node.StringAnswerValue ===
+        "decision-and-credit-decision-define-amount"
```

Comment:
Does this maybe need optional chaining in case some of it is undefined?

Reply:
Because the work item can't be skipped it will be always around and the question is required so the answer will also always exist